README.md

```diff
@@ -16,40 +15,39 @@ branch on which this work is based:
 ```
 git clone https://github.com/DataWaveProject/CAM.git
 cd CAM
-git checkout datawave_ml
+git checkout coupling
 ```
 This branch is built upon the `cam6_3_139` tag from the
 [main ESMCOMP/CAM repository](https://github.com/ESCOMP/CAM).
 
-### Preparing FTorch and setting the CIME component in CESM
+### Obtaining `FTorch`
 
-We also need to build and link _FTorch_ which will allow us to use our PyTorch-based
-neural nets in CAM. This has two steps.
+To use PyTorch-based neural nets in CAM, we first need to build and link `FTorch`.
 
-#### Building FTorch
+To install `FTorch` on Derecho follow the instructions in section [`FTorch` on Derecho](#ftorch-on-derecho) below.
```

Comment:
Do we want to add a placeholder section for building on CSD3 here now, or do we cross that bridge when we come to it?

Reply:
Good idea, but I don't think we should add them until we have the instructions ready to go.

We should also consider/discuss who these instructions are for. My understanding is it's for users who will most likely only run on `derecho` or other NCAR systems. So it might add confusion.

Either way, I do agree that we should write up `CSD3` instructions, but I am not sure yet, where they would best belong -- perhaps we can discuss in our next internal meeting.

Reply:
I agree.
I am hoping that I will have time in January to work on FTorch-CESM-CSD3 and make this overall process much simpler, but for now we should leave it until we have something to write about. To my mind documentation to run on CSD3 would go into the CSD3 documentation pages alongside more general docs about running CESM on CSD3.

I would also second Tom's point that this is intended as a resource specifically for our DataWave partners who run on Derecho at this point. We should leave it as specific to them, and adapt as necessary should they need to run on other systems.